docs/configuration/pipeline-configuration-files.mdx

```diff
@@ -1,6 +1,5 @@
 ---
 title: 'Pipeline Configuration Files'
-slug: 'pipeline-configuration-files'
```

Comment:
Default I guess?

Reply:
Yes, the default is the filename. I think it's simpler if we just use that, no need to open the file to know how it needs to be addressed.

Reply:
What I like about explicitly setting slugs is that no "docs refactoring" will affect URLs. But what you said makes sense too.

Reply:
Well there's always pro's and con's 😄